docs/configuration.md

```diff
@@ -22,10 +22,12 @@ header: true # Set to false to hide the header
 footer: '<p>Created with <span class="has-text-danger">❤️</span> with <a href="https://bulma.io/">bulma</a>, <a href="https://vuejs.org/">vuejs</a> & <a href="https://fontawesome.com/">font awesome</a> // Fork me on <a href="https://github.com/bastienwirtz/homer"><i class="fab fa-github-alt"></i></a></p>' # set false if you want to hide it.
 
 columns: "3" # "auto" or number (must be a factor of 12: 1, 2, 3, 4, 6, 12)
+vlayout: true # default to the vertical layout
 connectivityCheck: true # whether you want to display a message when the apps are not accessible anymore (VPN disconnected for example)
 
 # Optional theming
 theme: default # 'default' or one of the theme available in 'src/assets/themes'.
+theme_use_dark: false # true or false, useful for overriding browser default in new sessions
```

Comment:
As the related settings is named `colors`, I see something like that: 
```yaml
default_colors: dark # 'light', 'dark', or 'auto' (default, base on browser preference) 
```
It's also a bit more explicit (I like when the config speaks for itself, with little documentation necessary)